support-payment-api/src/main/resources/riff-raff.yaml

```diff
@@ -10,7 +10,7 @@ deployments:
     parameters:
       amiParameter: AMIPaymentapi
       amiTags:
-        Recipe: jammy-membership-java8
+        Recipe: jammy-membership-java
```

Comment:
Based on https://amigo.gutools.co.uk/recipes, I would’ve expected this to be [jammy-membership-java11](https://amigo.gutools.co.uk/recipes/jammy-membership-java11). I assume I’ve missed something, since this works in CODE – do you know what?

Reply:
Ah perfect, that explains it!